ecs-agent/credentials/manager.go

```diff
@@ -0,0 +1,160 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
```

Comment:
is this being removed from `agent/credentials/`?

Reply:
Yes that's right. This PR includes that change.

Reply:
![Screenshot 2023-05-11 at 12 19 07 PM](https://github.com/aws/amazon-ecs-agent/assets/9619772/f681cea4-b022-41c0-bf31-b5eb4a5212d5)
Git thinks that the file was moved to agent/vendor.